### ISSUE TITLE
move some global cli options to be output specific

### ISSUE THREAD

**Issue by dericed** [mipops/dvrescue#801]
Now that we can support >1 output, some options should be output specific:

```
    --merge-output-speed
    --merge-ignore-speed
    --merge-output-concealed
    --merge-ignore-concealed
```

For instance:

`dvrescue ...  --merge-output-speed --merge-output-concealed -m - --merge-ignore-speed --merge-ignore-concealed file.dv` should output all frames to the pipe (to show in a player) while frames at non-standard speed and fully-concealed are not written to the output file.

**Comment by g-maxime** [comment]
Fixed by #810

**Comment by dericed** [comment]
thx :D
